shuup/front/templates/shuup/front/dashboard/dashboard.jinja

```diff
@@ -0,0 +1,38 @@
+{% extends "shuup/front/base.jinja" %}
+{% block title %}{% trans %}My Dashboard{% endtrans %}{% endblock %}
+
+{% block content_title %}{% trans %}Dashboard{% endtrans %}{% endblock %}
+
+{% block breadcrumb %}
+    {% call macros.breadcrumb() %}
+        {{ macros.render_breadcrumb_item("/", _("Home")) }}
+        {{ macros.render_breadcrumb_item(None,  _("Dashboard"), True) }}
+    {% endcall %}
+{% endblock %}
+
+{% block content %}
+<div class="container customer-dashboard">
+    <div class="row">
+        <div class="col-md-3 section-navigation">
+            <div class="sidebar-list">
+                <a class="sidebar-list-item{% if request.path == '/dashboard/' %} active{% endif %}" href="{{ url("shuup:dashboard") }}"><i class="fa fa-tachometer"></i> {% trans %}Dashboard{% endtrans %}</a>
```

Comment:
Is this `request.path == '/dashboard/'` same as there is no item in context? Point beign that these comparisons to path looks sketcy.